src/methods/delta/postDeltaOrder.ts

```diff
@@ -0,0 +1,44 @@
+import { API_URL } from '../../constants';
+import type { ConstructFetchInput } from '../../types';
+import { DeltaAuctionOrder, ParaswapDeltaAuction } from './helpers/types';
+
+export type DeltaOrderToPost = {
+  partner?: string; // @TODO not sure if this is optional, need to check
```

Comment:
It is indeed optional, just checked
`'anon'` is assigned if not specified during /post